frontend/src/app/organization/organization.component.html

```diff
@@ -14,8 +14,6 @@
   [isUserOrgMember]="isUserOrgMember$ | async">
 </volontulo-organization-offers-list>
 </div>
-<ng-template #noOrganization>
-  <div class="container mt-3">
-    <h3>Nie znaleziono organizacji.</h3>
-  </div>
+<ng-template #notFound>
+  <volontulo-page-404></volontulo-page-404>
```

Comment:
If we want to have 100% generic 404 page, then I don't think we should repeat 404 template in every view. Anyway, some more "customized" not found pages might be okay (like i.e. "organization not found" with a link to organization search - or organization list, as there's no search yet).

Reply:
I've talk with @w1stler about that and he've said that it should always look the same, so maybe we should discuss that on Wednesday. :)